Engine/source/gui/controls/guiMLTextCtrl.cpp

```diff
@@ -768,7 +768,7 @@ void GuiMLTextCtrl::onMouseDown(const GuiEvent& event)
    mSelectionAnchorDropped = event.mousePoint;
    if (mSelectionAnchor < 0)
       mSelectionAnchor = 0;
-   else if (mSelectionAnchor >= mTextBuffer.length())
+   else if (mSelectionAnchor >= mTextBuffer.length() - 1)
```

Comment:
I can't remember what issue I was having with this, but I remember it causing some bug. When reading the code it makes sense that it should be what it was. But something was happening. I will take a look and talk with other people that worked with our code.